lib/tts_isolate.dart

```diff
@@ -48,28 +57,56 @@ Future<void> isolateMain(
           .limit(1)
           .snapshots()
           .listen((latestMessage) async {
-        if (latestMessage.docs.isNotEmpty) {
-          final message = latestMessage.docs[0];
-          if (message.data().containsKey('type')) {
-            final type = message['type'] as String?;
+        final docs = latestMessage.docs;
+        if (docs.isNotEmpty) {
+          final message = docs.first;
+          final messageData = message.data();
+          if (messageData.containsKey('type')) {
+            final type = messageData['type'] as String?;
             switch (type) {
               case "message":
-                final textToSpeak = message['message'] as String?;
-                String finalMessage = '';
-                if (textToSpeak != null) {
-                  final userName = message['author']['displayName'] as String?;
-                  finalMessage = isPreludeMuted
-                      ? textToSpeak
-                      : (userName != null
-                          ? '$userName said $textToSpeak'
-                          : textToSpeak);
+                final messageModel = TwitchMessageModel(
+                    messageId: message.id,
+                    author: TwitchUserModel(
+                      userId: messageData['tags']['user-id'],
+                      login: messageData['author']['displayName'],
+                    ),
+                    message: messageData['message'],
+                    reply: messageData['reply'] != null
+                        ? TwitchMessageReplyModel(
+                            messageId: messageData['reply']['messageId'],
+                            message: messageData['reply']['message'],
+                            author: TwitchUserModel(
+                              userId: messageData['reply']['userId'],
+                              displayName: messageData['reply']['displayName'],
+                              login: messageData['reply']['userLogin'],
+                            ),
+                          )
+                        : null,
+                    tags: messageData['tags'],
+                    annotations: TwitchMessageAnnotationsModel.fromMap(
+                        messageData['annotations']),
+                    thirdPartyEmotes: await getEmotes(Channel(
+                        messageData['channelId'].toString().split(":")[0],
+                        messageData['channelId'],
+                        messageData['author']['displayName'])),
```

Comment:
```suggestion
                    thirdPartyEmotes: {},
```

Ah, this is a bit unfortunate, we shouldn't fetch emotes like this repeatedly every channel as it will use too much data. The good news is that we don't really need third-party emotes for tts so you can wipe the map and pass an empty map.